tests/tasks/setup_ipa.yml

```diff
@@ -2,7 +2,7 @@
 - name: Clone ansible-freeipa repo
   git:
     repo: https://github.com/freeipa/ansible-freeipa
-    version: v0.1.10
+    version: v0.1.12
```

Comment:
I'm curious. In the future, if you update the certificate role and the latest ansible-freeipa version had been also upgraded then, you have to bump it manually to test against the latest like this? If it's not automated, could there be any chance to miss some unfortunate combinations? Or it's not an issue?

Reply:
Correct @nhosoi. If freeipa project implement something new there might be updates in ansible-freeipa as well in order to get this feature available. In that case we might need to bump this version.

If we miss that the ipa tests will start to fail and we'd need to update that number. That could happen specially when there are major distro versions released (for example CentOS 8). 

I chose to use the fixed version instead of using git master because, I believe, the instabilities caused by eventual need for updates will be less frequent then the instabilities caused by in-development code.

At this time the update wasn't really required. I've just bumped version to leave it up-to-date.